.pre-commit-config.yaml

```diff
@@ -11,14 +12,18 @@ repos:
     hooks:
     # Run the linter:
     -   id: ruff-check
-        name: ruff
+        name: ruff check
+        language: python
     # Run the formatter:
     -   id: ruff-format
+        name: ruff format
+        language: python
```

Comment:
```suggestion
```
There’s no “additional_dependencies” here, so explicitly specifying the language doesn’t provide any benefit.